src/platform_impl/windows/mod.rs

```diff
@@ -65,6 +68,72 @@ impl WindowId {
     }
 }
 
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct OsSpecificWindowEvent {
+    window: HWND,
+    msg: UINT,
+    wparam: WPARAM,
+    lparam: LPARAM,
+    retval: *mut Option<LRESULT>,
+}
+
+impl OsSpecificWindowEvent {
+    /// The message identifier of the window event (`WM_*`).
+    #[inline]
+    pub fn message(&self) -> u32 {
+        self.msg
+    }
+
+    /// The first parameter of the window event (`WPARAM`).
+    #[inline]
+    pub fn wparam(&self) -> usize {
+        self.wparam
+    }
+
+    /// The second parameter of the window event(`LPARAM`).
+    #[inline]
+    pub fn lparam(&self) -> isize {
+        self.lparam
+    }
+
+    /// Mark the current event processed, returning specified value
```

Comment:
Mark -> Marks
returning -> returning the

The value will be interpreted according to the meaning of the message. -> How the value is interpreted is dependent on the message's type.

It is undefined behavior to call this function unless during the event loop processing this corresponding event. -> ```Calling this function outside this event's call to [`event_handler`] will result in undefined behavior.```

Then append:
```[`event_handler`]: ../I/don't/know/the/proper/address/:)/../struct.EventLoop.html#method.run_whatever_the_function_name_was```

Reply:
done. a few more lines are added. ptal again.